### PR TITLE
Fixes

### LINKED ISSUE #1077: Docstrings suggest kwargs are available
The current documentation for SquaredExponential (http://openturns.github.io/openturns/master/user_manual/_generated/openturns.SquaredExponential.html) reads:

> **Available constructors**: 
>     SquaredExponential(*spatialDim*=1) 
>     SquaredExponential(*scale*)
>     SquaredExponential(*scale*, *amplitude*)

Apparently, this is *meant* to indicate that the SquaredExponential constructor accepts an integer argument that indicates the dimension of the covariance function inputs, and the default value of this argument is 1. However, there's an ambiguity, because unlike C++, Python functions may accept keyword arguments, so an expression like `spatialDim = 1` not only implies a default value of `spatialDim`, but seems to suggest that it is a keyword argument as well.

Yes, that the documentation for SquaredExponential begins with `SquaredExponential(*args)` rather than `SquaredExponential(*args, **kwargs)` might imply otherwise, but that's subtle and very easy to miss.

A possible fix to the documentation may be something roughly like this:

> **Available constructors**: 
>     SquaredExponential(*scale_or_spatial_dim*)
>     SquaredExponential(*scale*, *amplitude*)
>     SquaredExponential()
>
> **Parameters**:
> * *scale_or_spatial_dim*: Either a sequence of floats or an integer. If a sequence of floats, then it represents a vector of scale coefficients $\theta\in\mathbb{R}^n$. If an integer, then it represents the *length* of the vector of scale coefficients.
> * *amplitude*: ... same as before ...
>
> If the SquaredExponential constructor is called with no arguments, that is equivalent to the case where *scale_or_spatial_dim* equals 1.

### LINKED ISSUE #1241: Wishart.getCovariance segfault
The `Wishart::getCovariance` methods segfaults
```
import openturns as ot
d = ot.Wishart(ot.IdentityMatrix(6),12.5)
print(d.getCovariance())
```

cf http://trac.openturns.org/ticket/940

### LINKED ISSUE #1631: Missing image
@KieranDelamotte, it seems that in the example below the defined thumbnail_path = '../_static/pandas.png' isn't working. Missing image or is the path incorrect? 
[Interaction with Pandas](https://openturns.github.io/openturns/master/auto_data_analysis/manage_data_and_samples/plot_sample_pandas.html#sphx-glr-auto-data-analysis-manage-data-and-samples-plot-sample-pandas-py)

### PR DESCRIPTION
**Pull request by josephmure**
- [Fix misleading SquaredExponential ctor doc and misleading BoxCoxTransform ctor doc.](https://github.com/openturns/openturns/commit/8a338768b6245cf4b6947dd3493d9f79fb10ed88) Closes #1077 
- [DurbinWatson: state secondSample must be ordered](https://github.com/openturns/openturns/commit/46127d48bd02b9af5113463a94a53d07cb81b026) Closes https://github.com/openturns/openturns/issues/1199
- [SpaceFillingMinDist doc mentions scaling](https://github.com/openturns/openturns/commit/a16ee53e2dd5e5e23ff0e023ff4366d3978e2c6b) Closes https://github.com/openturns/openturns/issues/1203
- Pandas logo appears as thumbnail of the "Link Pandas and OpenTURNS" example. Closes #1631 
- [Fix Wishart and InverseWishart covariance matrix](https://github.com/openturns/openturns/pull/2746/commits/9226545291663e66986b262d44141f1e7626e120). Fixes #1241

### CODE REVIEW

File: lib/src/Uncertainty/Distribution/openturns/Wishart.hxx [hunk 1]
```diff
@@ -124,6 +124,9 @@ private:
   /** Compute the mean of the distribution */
   void computeMean() const override;
 
+  /** Compute the covariance of the distribution */
+  void computeCovariance() const override;
```
Comment (jschueller) [inline lib/src/Uncertainty/Distribution/openturns/Wishart.hxx:128]: needs some test

**Review by jschueller — APPROVED** [review]

File: lib/src/Uncertainty/Distribution/openturns/Wishart.hxx (hunk 1 above)
Comment (sofianehaddad) [reply lib/src/Uncertainty/Distribution/openturns/Wishart.hxx:128]: Absolutely! tests for the covariance

Comment (josephmure) [reply lib/src/Uncertainty/Distribution/openturns/Wishart.hxx:128]: Done

File: python/src/LinearModelTest_doc.i.in [hunk 2]
```diff
@@ -303,7 +303,7 @@ Parameters
 firstSample : 2-d sequence of float
     First tested sample.
 secondSample : 2-d sequence of float
-    Second tested sample, of dimension 1.
+    Second tested sample, of dimension 1. This sample should be ordered.
```
Comment (josephmure) [inline python/src/LinearModelTest_doc.i.in:306]:
```suggestion
    Second tested sample, of dimension 1. This sample should be ordered in increasing order.
```

**Review by sofianehaddad — APPROVED** [review]

**Pull request merged**